qiskit/qpy/binary_io/schedules.py

```diff
@@ -169,10 +169,10 @@ def _read_symbolic_pulse(file_obj, version):
 
         # And warn that this will change in future releases:
         warnings.warn(
-            "Complex amp support for symbolic library pulses will be deprecated. "
+            "Complex amp support for symbolic library pulses will be deprecated in Qiskit 1.0.0. "
```

Comment:
I think that the "will be deprecated" part was suited for the pending deprecation warning, but once we promote it to a deprecation warning then the support is deprecated, so I suggest to change the message a bit to reflect that:

```suggestion
            "Complex amp support for symbolic library pulses are deprecated, and will be removed in Qiskit 1.0.0. "
```
Similarly, on the sentence below (cannot give suggestions directly because the code wasn't changed), I would remove the "Once deprecated" part, because they are already deprecated (so we can say directly that `library pulses loaded from...will be converted automatically to...`), or change it to "once removed" if this doesn't happen yet but is planned for 1.0.

Reply:
You are obviously right. Thanks.